src/phases/move-effect-phase.ts

```diff
@@ -219,6 +211,8 @@ export class MoveEffectPhase extends HitCheckPhase {
         ? this.playMoveAnim(user)
         : Promise.resolve();
 
+    console.log(`hitCount: ${user.turnData.hitCount}, hitsLeft: ${user.turnData.hitsLeft}`);
+
```

Comment:
```suggestion
    console.log(`hitCount: ${user.turnData.hitCount}, hitsLeft: ${user.turnData.hitsLeft}`);

```

unless it's been left on purpose